scheduler/interfaces.go

```diff
@@ -0,0 +1,6 @@
+package scheduler
+
+type runCrontabInterface interface {
```

Comment:
IIRC, we created this interface was to allow us to mock it in tests, but we haven't. I suggest we either write some tests that mock it, or remove this interface from this PR.